client/my-sites/site-settings/form-analytics.jsx

```diff
@@ -198,4 +234,31 @@ export default protectForm( React.createClass( {
 		// Only show Google Analytics for business users.
 		return this.form();
 	}
-} ) );
+} );
+
+const mapStateToProps = ( state, ownProps ) => {
+	const { site } = ownProps;
+
+	const isGoogleAnalyticsEligible = site && site.plan && hasBusinessPlan( site.plan );
+	const jetpackModuleActive = isJetpackModuleActive( state, site.ID, 'google-analytics' );
+	const jetpackVersionSupportsModule = isJetpackMinimumVersion( state, site.ID, '4.5-beta1' );
+	const googleAnalyticsEnabled = site && (
+		! site.jetpack ||
+		( site.jetpack && jetpackModuleActive && jetpackVersionSupportsModule && isEnabled( 'jetpack/google-analytics' ) )
+	);
+
+	return {
+		selectedSite: site,
```

Comment:
Is this supposed to be coming down from above?

I'm not sure if you modified or added this, but in general I prefer not passing down entire site objects, instead either passing down a few required props from the site object or passing a site id and using the site selector here.

One such reason is that `getSelectedSite` can return `null`, such as when we're in the site selector or no site is in the URL.